frontend/src/components/form/base/ColorPicker/ColorSwatch.vue

```diff
@@ -17,7 +19,7 @@ import { contrastColor } from '@/common/helpers/colors.js'
 export default {
   name: 'ColorSwatch',
   props: {
-    color: { type: String, required: true },
+    color: { type: String, default: null },
```

Comment:
this is because at initialization it is null?

Reply:
Correct